apps/editors/helpers.py

```diff
@@ -220,6 +221,28 @@ def reviewers_score_bar(context, types=None, addon_type=None):
                                          addon_type=addon_type)))
 
 
+@register.inclusion_tag('editors/includes/files_view.html')
+@jinja2.contextfunction
+def all_distinct_files(context, version):
+    '''
+    Returns [file_ref, [Windows / Mac OS X]] if the same file for
+    Windows and Mac OS X was uploaded.
```

Comment:
Right, it's not returning anything.

Nevertheless, I throw away the hashes before I pass it to the context, see https://github.com/mozilla/olympia/pull/622/files#diff-c88d74d35db30ad465900047ea851404R239 where is just take the values of the array.